core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/CloneWorkspaceContainerRunner.scala

```diff
@@ -105,14 +105,20 @@ class CloneWorkspaceContainerRunner(
       cloneFail(workspaceId, "Cloning workspace resource container failed").map(_ => Complete)
   }
 
+  /**
+    * If the workspace still exists, updates the clone completed time on the workspace. Otherwise does nothing.
+    *
+    * @param wsId the ID of the workspace
+    * @param finishTime the time cloning completed
+    * @return the number of records updated, wrapped in a Future
+    */
   def cloneSuccess(wsId: UUID, finishTime: DateTime)(implicit
     executionContext: ExecutionContext
-  ): Future[Option[Workspace]] =
+  ): Future[Int] =
     getWorkspace(wsId).flatMap {
-      case Some(workspace) =>
-        val updatedWS = workspace.copy(completedCloneWorkspaceFileTransfer = Some(finishTime))
-        dataSource.inTransaction(_.workspaceQuery.createOrUpdate(updatedWS)).map(Option(_))
-      case None => Future.successful(None)
+      case Some(_) =>
+        dataSource.inTransaction(_.workspaceQuery.updateCompletedCloneWorkspaceFileTransfer(wsId, finishTime.toDate))
```

Comment:
We are obviously lacking coverage beyond the fact that `cloneSuccess` is called (in a unit test), but coverage will be added via the e2e I am working on.